homeassistant/components/light/mochad.py

```diff
@@ -26,6 +26,7 @@
         vol.Optional(CONF_NAME): cv.string,
         vol.Required(CONF_ADDRESS): cv.x10_address,
         vol.Optional(mochad.CONF_COMM_TYPE): cv.string,
+        vol.Optional(mochad.CONF_MAX_XDIM_VALUE): cv.byte,
```

Comment:
I think if we know all values, we should validate it here like `vol.In([63, 255])`. Also with default value